test/data/test_functional.py

```diff
@@ -160,6 +161,19 @@ def test_RegexTokenizer(self):
         self.assertEqual(jit_tokens, eager_tokens)
         self.assertEqual(jit_tokens, ref_results)
 
+    def test_NgramsTransform(self):
+        test_sample = ['here', 'we', 'are']
+        ref_results = ['here', 'we', 'are', 'here we', 'we are', 'here we are']
```

Comment:
Another option is to test again `ngrams_iterator` [here](https://github.com/pytorch/text/blob/6fcab35e80717a666d6ccf6b2fbcd7fec684db28/torchtext/data/utils.py#L205)

Reply:
Added this!